case_studies/redshift/train_rs.py

```diff
@@ -0,0 +1,191 @@
+# Author: Qiaozhi Huang
+# train to predict redshift using network
+# python case_studies/redshift/train_rs.py --resume-model=/home/qiaozhih/bliss/case_studies/redshift/training_runs/00017-runreg/000000_model.pt --resume-opt=/home/qiaozhih/bliss/case_studies/redshift/training_runs/00017-runreg/000000_opt.pt --nick=reg
+import json
+import os
+import re
+
+import click
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from network_rs import LitRegressor, Regressor
```

Comment:
Would you merge train_rs.py and network_rs.py into a single file, to make it easier for others to understand what code in this directory are related? And also name the file something like `photoz_from_fluxes.py` (network_rs is somewhat cryptic).